renovate.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["config:recommended"],
+  "packageRules": [
+    {
+      "matchPackageNames": ["node", "actions/node-versions", "pnpm"],
+      "enabled": false
+    },
+    {
+      "matchPackagePatterns": ["@types/node", "@nomicfoundation/.*", "chai", "eslint"],
+      "matchUpdateTypes": ["major"],
+      "enabled": false
+    },
+    {
+      "matchUpdateTypes": ["patch", "minor"],
+      "schedule": ["before 10am on the first day of the month"],
+      "groupName": "other minor updates"
+    },
+    {
+      "matchUpdateTypes": ["major"],
+      "schedule": ["before 10am on the first day of the month"],
+      "groupName": "other major updates"
+    },
+    {
+      "matchDepTypes": ["dependencies"],
+      "matchUpdateTypes": ["patch", "minor"],
+      "schedule": ["before 1am on the first day of the month"],
+      "groupName": "dependencies (non-major)"
+    },
+    {
+      "matchPackagePatterns": ["@api3/.*"],
+      "matchUpdateTypes": ["major"],
+      "schedule": ["before 6pm"],
+      "groupName": "api3 major updates"
+    },
+    {
+      "matchPackagePatterns": ["@api3/.*"],
+      "matchUpdateTypes": ["patch", "minor"],
+      "schedule": ["before 6pm"],
+      "groupName": "api3"
+    }
+  ],
+  "rangeStrategy": "bump",
+  "lockFileMaintenance": {
+    "enabled": false
+  },
+  "baseBranches": ["main"],
+  "reviewers": ["Siegrift", "acenolaza"],
```

Comment:
Let's also add it to the actively maintained repos in our docs.

Reply:
You mean [here](https://github.com/api3dao/api3-docs/blob/main/docs/dapps/integration/api3-contracts.md#L17)?

Reply:
Here -> https://github.com/api3dao/tasks/blob/a934d98952f59f7d2d0817663f6ff3977ee01612/Technical%20management/Tasks%20documentation/activelyMaintainedRepos.md#L4